<launch>

  <group ns="safety">
    <node pkg="math_pkg" name="safety_server" type="safety"/>
  </group>

  <group ns="ik">
    <node pkg="math_pkg" name="ik_server" type="ik"/>
  </group>

  <group ns="cost">
    <node pkg="math_pkg" name="cost_server" type="cost"/>
  </group>

</launch>